CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.21.7][] - 2020-02-14
```

Comment:
These versions are auto generated with yarn release. We change the changelog under the Unreleased section

Reply:
So I just remove `## [0.21.7] ......` ?

Reply:
Yep